lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1722,7 +1722,7 @@ def api_client
       else
         unless @client.authorization.expired?
           begin
-            @client.authorization.fetch_access_token!
+            @client.authorization.fetch_access_token!(:use_configured_scope => true)
```

Comment:
Have you tested with the gRPC path? Do you need an equivalent change there (lines 1717-1718)?

Reply:
Also, please fix the `rubocop` errors.

Reply:
Where would I need to put the use_grpc = true to test with it? (Assuming a regular google-fluentd package.)

Reply:
`rubocop` now passes.

Reply:
You can add that option (`use_grpc true`, no `=`) in the `google_cloud` output plugin stanza.

Reply:
So with that it doesn't work again. That provisions the updater function provided by googleauth. Unfortunately the one it monkey patches into Signed::OAuth2::Client does not take any options. I pushed a workaround in this library as it doesn't seem to be too odd to do the request mangling here. I guess rubocop will tell me if the coding style matches.